activiti-core/activiti-bpmn-model/src/main/java/org/activiti/bpmn/model/LinkEventDefinition.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2010-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.bpmn.model;
+
+import java.util.List;
+
+public class LinkEventDefinition extends EventDefinition{
+
+    protected String id;
+
+    protected String name;
+
+    protected String target;
+
+    protected List<String> sources;
+
+    public LinkEventDefinition clone() {
+        LinkEventDefinition clone=new LinkEventDefinition();
```

Comment:
Please, add spaces before and after `=` operator

Reply:
Done